app/src/Wizard.tsx

```diff
@@ -0,0 +1,116 @@
+export function Wizard () {
+  const [contractTrait, setContractTrait] = useState('Lottery')
+  const [contractName, setContractName] = useState('MyContract')
+  const [contractParams, setContractParams] = useState([])
+  const [author, setAuthor] = useState('')
+  const [license, setLicense] = useState('')
+  const [showInvokeModal, setShowInvokeModal] = useState(false)
+  const [modalTitle, setModalTitle] = useState('')
+  const [modalBody, setModalBody] = useState('')
+
+  const codeGen = new CodeGen()
+
+  function handleClick (type: string) {
+    if (type === 'Download') {
+      setModalTitle('Download')
+      setModalBody('Not implemented yet')
+      setShowInvokeModal(true)
+    }
+    if (type === 'Copy') {
+      const code = codeGen.getCode()
+      void navigator.clipboard.writeText(code)
+    }
+    if (type === 'Deploy') {
+      setModalTitle('Deploy')
+      setModalBody('Not implemented yet')
+      setShowInvokeModal(true)
+    }
+    if (type === 'Invoke') {
+      const invokeGen = new InvokeCommandGen()
+      setModalTitle('Invoke contract')
+      setModalBody(
+        invokeGen.generateInvokeCommand(
+          contractTrait,
+          contractName,
+          contractParams
+        )
+      )
+      setShowInvokeModal(true)
+    }
+    if (type === 'Open') {
+      setModalTitle('Open in Pulsar')
+      setModalBody('Not implemented yet')
+      setShowInvokeModal(true)
+    }
+    if (type === 'Compile') {
+      setModalTitle('Compile')
+      setModalBody('Not implemented yet')
+      setShowInvokeModal(true)
+    }
+  }
+  const handleInvokeModalClose = () => {
+    setShowInvokeModal(false)
+  }
+
+  return (
+    <div className="Wizard">
+      <Navbar currentPage='/wizard'/>
+        <div className="row flex-grow-1">
+          <div className="col-3">
+            <Toolbox
+              contractName={contractName}
+              onContractNameChanged={setContractName}
+              contractTrait={contractTrait}
+              onContractTraitChanged={setContractTrait}
+              author={author}
+              onAuthorChanged={setAuthor}
+              license={license}
+              onLicenseChanged={setLicense}
+              updateParams={setContractParams}
+              handleClick={handleClick}
+            />
+          </div>
+          <div className="col-8">
+            <Editor
+              contractTrait={contractTrait}
+              contractName={contractName}
+              author={author}
+              license={license}
+              codeGen={codeGen}
+            />
+          </div>
+          <div
+            className="modal show"
+            style={{ display: 'block', position: 'initial' }}
+          >
+            <Modal show={showInvokeModal} onHide={handleInvokeModalClose}>
+              <Modal.Header closeButton>
+                <Modal.Title>{modalTitle}</Modal.Title>
+              </Modal.Header>
+              <Modal.Body>
+                <p>
+                  <pre>
```

Comment:
As a nitpick, would it be possible to add a more "terminal-like" style to the invoke modal ? Currently is in this way:

![image](https://github.com/eigerco/nebula/assets/5852187/26ce4aba-9bcb-4ba5-bc4a-6d1206dd0101)

By adding this style property  (playing around):

```html
<pre style={{ backgroundColor: 'black' }}>
``` 
I was able to get this :) :

![image](https://github.com/eigerco/nebula/assets/5852187/ff9dfb8c-23e0-4b6d-8fe9-96193863e144)

I like it more let me know what do you think. Also, i am wondering if the typical button of "copy to clipboard" would be easy to add. 

Thanks for the great work :heart_eyes:

Reply:
I can change it. As for the button - it is there and it works :)

Reply:
Ah yeah, you mean this one right ? for copying the contract code:

![image](https://github.com/eigerco/nebula/assets/5852187/9d95004e-3cac-4a7b-9927-eef144992647)

I am suggesting to add the same, but in the invoke modal. So the user can copy the CLI invokation also :)

Reply:
Ah ok :)

Reply:
Well, maybe needs a bit more of elaboration as, with the code i [shared before](https://github.com/eigerco/nebula/pull/14#discussion_r1262368862), affects every modal ofc .....

![image](https://github.com/eigerco/nebula/assets/5852187/64edeb43-90c0-4c59-89d1-a2fa807fac04)

Reply:
Added both the background and button.

Reply:
Great, thanks! :)